sky/tools/create_ios_framework.py

```diff
@@ -9,20 +9,14 @@
 
 import argparse
 import os
-import platform
-import shutil
-import subprocess
 import sys
 
 from create_xcframework import create_xcframework  # pylint: disable=import-error
-
-ARCH_SUBPATH = 'mac-arm64' if platform.processor() == 'arm' else 'mac-x64'
-DSYMUTIL = os.path.join(
-    os.path.dirname(__file__), '..', '..', 'buildtools', ARCH_SUBPATH, 'clang', 'bin', 'dsymutil'
+from sky_utils import (  # pylint: disable=import-error
```

Comment:
Is there any way to do these imports in a way that requires retaining the library prefix (like `sky_utils.`) on usages?

Reply:
Yeah I could just import as `import sky_tools` and use the prefixed names everywhere. The list is kind of long at this point.

Reply:
Done.